gp-nested-forms/gpnf-triggered-population.php

```diff
@@ -119,6 +121,8 @@ public function output_script() {
 								window.gpnf_triggered_population_entry = response.data
 								GPNestedForms.loadEntry( response.data );
 							}
+							// re-enable the field after entry creation is processed
+							$( $( '#field_{0}_{1}'.format( self.formId, self.triggerFieldId ) ).attr( 'disabled', false ) );
```

Comment:
You're double-wrapping the `$()` here (and above), which isn't needed.

I can also see `$( '#field_{0}_{1}'.format( self.formId, self.triggerFieldId )` being moved to a `var` being helpful at this point 😄